schema@v1.3.0/row-meta-schema.json

```diff
@@ -77,6 +77,10 @@
           "type": "string",
           "format": "uri-reference"
         },
+        "crs": {
+          "description": "Coordinate reference system",
+          "$ref": "./schema@v1.3.0#/definitions/crs"
+        },
```

Comment:
Kan je deze ook ergens groeperen, dat dit alleen bij type geometry nuttig is?

Reply:
Je mag alleen een crs toevoegen bij een geometry veld. Dat wordt afgedwongen door de "not": "crs" regel.

Reply:
nie!